### PR TITLE
FIx spelling and grammar, also missing doc of level,pp_raw

### PR DESCRIPTION
**Pull request by Maverun**

### CODE REVIEW

File: osuapi/model.py [hunk 1]
```diff
@@ -118,15 +118,19 @@ class User(AttributeModel):
...
-        Total sum of the best scores fro all the ranked beatmaps played online.
+        Total sum of the best scores from all the ranked beatmaps played online.
     total_score : int
         Total sum of all scores on ranked beatmaps, including failed trails.
     pp_rank : int
         Global ranking place.
+    level: float
+        User's level
+    pp_raw: float
+        User's Performance Point
```
Comment (khazhyk) [inline osuapi/model.py]: performance points

**Comment by Maverun** [comment]
fixed.

**Comment by khazhyk** [comment]
Can you squash?

**Comment by Maverun** [comment]
How do.. :| 
I am pretty noob with github...

**Comment by khazhyk** [comment]
Rebased https://github.com/khazhyk/osuapi/commit/c8a9b4497f2c5008d44b2b843b9d2da990ce2af1

You can squash commits with an interactive rebase. e.g. https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/

**Pull request closed without merging**